src/core/public/overlays/utils.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useRef } from 'react';
+import ReactDOM from 'react-dom';
+import { MountPoint } from 'kibana/public';
+import { I18nProvider } from '@kbn/i18n/react';
+
+/**
+ * Mount converter for react components.
+ *
+ * @internal
+ */
+export const mountForComponent = (component: React.ReactElement): MountPoint => (
+  element: HTMLElement
+) => {
+  ReactDOM.render(<I18nProvider>{component}</I18nProvider>, element);
+  return () => ReactDOM.unmountComponentAtNode(element);
+};
```

Comment:
The `core` version of `mountForComponent` is only used in test files. However, I did not see any naming convention for files that are not test but only used for tests in Kibana code base, so the method is in the here. Should it be put elsewhere (or duplicated in each test file instead ?)

Reply:
I've seen several conventions, one of the them is to create `test_helpers/` folder.